Wikipedia/Code/MWKDataStore.m

```diff
@@ -704,7 +711,7 @@ - (BOOL)migrateToCoreData:(NSError **)error {
 #pragma mark - Memory
 
 - (void)didRecievememoryWarningWithNotifcation:(NSNotification *)note {
```

Comment:
Oh, would you mind fixing the lower case `m` and 2 spelling errors in this method signature?

Should  be:
`didReceiveMemoryWarningWithNotification`